java/connecting-to-bigtable/README.md

```diff
@@ -0,0 +1,12 @@
+# Connecting to Cloud Bigtable
+
+This is a collection of samples relating to connecting to Cloud Bigtable.
```

Comment:
If there are instructions on how to run this sample (and not just the tests) on the cloud docs, please provide the link. If not please provide content here similar to 
https://github.com/GoogleCloudPlatform/cloud-bigtable-examples/tree/master/java/quickstart

Reply:
The current statement does not explain whether we're using the HBase client and what operations will be completed.

Reply:
Updated. 
I'm not sure if anyone would want to run this sample on it's own since it doesn't do anything beyond connecting to the instance. Correct me if I'm wrong though and I can add the information to run it like in the Quickstart.